pulmap.yml

```diff
@@ -6,10 +6,13 @@
     - site_vars.yml
     - group_vars/pulmap.yml
   roles:
-    - role: pulibrary.ruby
+    - role: pulibrary.ruby23
```

Comment:
I just deployed this branch of the playbook to staging, but with ruby 2.4 instead of 2.3. It works just fine. This role change isn't needed.